networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/GossipFailureLogger.java

```diff
@@ -25,48 +26,63 @@
 public class GossipFailureLogger {
   private static final Logger LOG = LogManager.getLogger();
 
+  private final boolean shouldSuppress;
   private final String messageType;
-  private UInt64 lastErroredSlot;
+  private Optional<UInt64> lastErroredSlot;
   private Throwable lastRootCause;
 
-  public GossipFailureLogger(final String messageType) {
-    this.messageType = messageType;
+  public GossipFailureLogger(final String messageType, final boolean shouldSuppress) {
+    this.messageType = shouldSuppress ? messageType + "(s)" : messageType;
+    this.shouldSuppress = shouldSuppress;
   }
 
-  public synchronized void logWithSuppression(final Throwable error, final UInt64 slot) {
+  public synchronized void logWithSuppression(
```

Comment:
I don't like the name, we don't know if supression is on

Reply:
ye i think i'll just rename the method to `log`